### PR TITLE
Support customization of link text

### PR DESCRIPTION
**Pull request by sobition**
UPDATE: added buttonText sample in script.js

UPDATE: updated styling of locate me button

UPDATE: added buttonText in the documentation examples

UPDATE: removed buttonText sample in script.js

### CODE REVIEW

File: src/L.Control.Locate.scss [hunk 1]
```diff
@@ -4,6 +4,10 @@
     font-size: 1.4em;
     color: #444;
     cursor: pointer;
+    width: fit-content !important;
+    min-width: 30px !important;
+    min-height: 30px !important;
```
Comment (domoritz) [inline src/L.Control.Locate.scss:9]: Can you try to not require the `!important` annotation? It usually leads to problems in other places.

File: src/L.Control.Locate.js [hunk 2]
```diff
@@ -282,7 +282,13 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
                 link.title = options.strings.title;
                 link.role = 'button';
                 link.href = '#';
-                var icon = L.DomUtil.create(options.iconElementTag, options.icon, link);
+                var icon;
+                if(options.strings.buttonText) {
```
Comment (domoritz) [inline src/L.Control.Locate.js:286]: Please make the formatting consistent with the rest of the code.

**Review by domoritz — CHANGES_REQUESTED** [review]
Thank you for the pull request.

**Comment by yanis-sadeg** [comment]
Hello, I would like if this request is accepted, Any new about it?

**Comment by domoritz** [comment]
I am waiting for updates to my comments. Please make a new pull request with the changes @Lunok.

**Pull request closed without merging**